pynest/nest/tests/test_siegert_neuron.py

```diff
@@ -57,7 +58,6 @@ def setUp(self):
         nest.ResetKernel()
         nest.SetKernelStatus(
             {'resolution': self.dt, 'use_wfr': False, 'print_time': True})
-
```

Comment:
To me it seems odd to to remove this line instead of (old) line 62. The same is true for line 75, where probably line 77 should be removed instead. Could you edit this manually and see if it still passes the sytle check?